src/ngraph/op/argmax.hpp

```diff
@@ -42,6 +42,13 @@ namespace ngraph
 
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
+
+            /// \return The default value for op. Used for zero dimension tensor elimination.
+            virtual std::shared_ptr<Node> get_default_value() const override
+            {
+                auto type = get_input_element_type(0);
+                return ngraph::make_constant_from_string("-INFINITY", type, get_shape());
```

Comment:
I guess there are two cases here:

1. The reduction axis does not have size zero. Then it doesn't matter what we put here, because the output tensor will have zero elements.
2. The reduction axis has size zero. Then I'm actually not sure what we should put! TensorFlow calls it an error.

For now let's fill it with zero, but we should probably add a check to `IndexReduction` to make sure the reduced axis does not have size zero.

Reply:
Same for ArgMin. (Whatever values we put here are meaningless anyway.)